lib/carrierwave/mongoid.rb

```diff
@@ -104,9 +105,126 @@ def serializable_hash(options=nil)
         end
       RUBY
     end
+
+    def mount_uploaders(column, uploader=nil, options={}, &block)
+      field options[:mount_on] || column, type: Array, :default => []
+
+      # super
```

Comment:
The problem seems to be here. If I uncomment it, it returns this error-
` super: no superclass method `mount_uploaders' for MongoUser:Class`